aws/nlb/nlb.tf

```diff
@@ -0,0 +1,260 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# VARIABLES / LOCALS / REMOTE STATE
+# ----------------------------------------------------------------------------------------------------------------------
+
+variable "vpc_id" {
+  type        = string
+  description = "VPC ID to associate with NLB"
+}
+variable "subnet_ids" {
+  type        = list(string)
+  description = "A list of subnet IDs to associate with NLB"
+}
+
+variable "internal" {
+  type        = bool
+  default     = false
+  description = "A bool flag to determine whether the NLB should be internal"
+}
+variable "http_ingress_cidr_blocks" {
+  type        = list(string)
+  default     = ["0.0.0.0/0"]
+  description = "List of CIDR blocks to allow in HTTP security group"
+}
+variable "http_ingress_prefix_list_ids" {
+  type        = list(string)
+  default     = []
+  description = "List of prefix list IDs for allowing access to HTTP ingress security group"
+}
+variable "domain_name" {
+  type        = string
+  default     = ""
+  description = ""
+}
+variable "dns_zone_id" {
+  type        = string
+  default     = ""
+  description = ""
+}
+variable "dns_zone_name" {
+  type        = string
+  default     = ""
+  description = ""
+}
+variable "certificate_arn" {
```

Comment:
We should probably review some of the variables here before merging. It seems that some of them are not in use